Good-Bids/goodbids
# [#503] Hooking up Share link with JS

# Summary

This hooks up the Share for free bids link with out copy link JS. I decided to create a new shortcode called `copy-link-btn` as it kept it all in one place to be used again on other front-end pages. I did some refactoring of the classes to the copy-link short code as well. 

## Issues

* #503

## Testing Instructions

1. Go to a live auction page. 
2. Click on the Share button and a popup info text should show "Copied"
3. Make sure the share link was copied. 

## Screenshots


![Feb-29-2024 12-59-16](https://github.com/Good-Bids/goodbids/assets/91974372/aabc94d9-da82-4b79-b3e2-272087543afa)
